ghcup-vanilla-0.0.8.yaml

```diff
@@ -4572,6 +4572,122 @@ ghcupDownloads:
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Mint:
+            < 20: &ghc984-x86_64-ubuntu1804
+              dlHash: e826990bd46a93ff8c87cf279fe1c480d763df4f1140fdf33b26d22ef3e21f06
+              dlOutput: ghc-9.8.4-x86_64-linux-ubuntu18_04.tar.xz
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu18_04-linux.tar.xz
+            '>= 20': &ghc984-x86_64-ubuntu2004
+              dlHash: d83941c9be977e7dc1a6f3eca32bc11e01a7cc3e6b678a226856a99e2ae22ce9
+              dlOutput: ghc-9.8.4-x86_64-linux-ubuntu20_04.tar.xz
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-ubuntu20_04-linux.tar.xz
+          Linux_RedHat:
+            '>= 8': &ghc984-x86_64-rocky8
+              dlHash: 7e10f00bdc9e552d9fc3d330b27681b340b0653df4741983b49937c2b533a5c4
+              dlOutput: ghc-9.8.4-x86_64-linux-rocky8.tar.xz
+              dlSubdir: ghc-9.8.4-x86_64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-rocky8-linux.tar.xz
+            unknown_versioning: *ghc984-x86_64-centos7
+          Linux_Ubuntu:
+            ( >= 16 && < 19 ): *ghc984-x86_64-ubuntu1804
+            unknown_versioning: *ghc984-x86_64-ubuntu2004
+          Linux_UnknownLinux:
+            unknown_versioning: *ghc984-x86_64-rocky8
+          Windows:
+            unknown_versioning:
+              dlHash: 184c9184d6439f1da8bca0214011ef6074fb5946eaa5840dc0c3e884515d7c52
+              dlOutput: ghc-9.8.4-x86_64-windows.tar.xz
+              dlSubdir: ghc-9.8.4-x86_64-unknown-mingw32
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-x86_64-unknown-mingw32.tar.xz
+        A_ARM64:
+          Darwin:
+            unknown_versioning:
+              dlHash: e2f12a922754fd28511512875bf6d9eb3e0cce7fc963a7266f6e1661aeabd7ed
+              dlOutput: ghc-9.8.4-aarch64-darwin.tar.xz
+              dlSubdir: ghc-9.8.4-aarch64-apple-darwin
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-aarch64-apple-darwin.tar.xz
+          Linux_UnknownLinux:
+            unknown_versioning:
+              dlHash: 15cf9341437af2c6f75bbaf8222824f5b51c32f0637ed054513f165999b0a87d
+              dlOutput: ghc-9.8.4-aarch64-linux-deb10.tar.xz
+              dlSubdir: ghc-9.8.4-aarch64-unknown-linux
+              dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-aarch64-deb10-linux.tar.xz
+      viChangeLog: https://downloads.haskell.org/~ghc/9.8.4/docs/users_guide/9.8.4-notes.html
+      viReleaseDay: '2024-11-27'
+      viSourceDL:
+        dlHash: 17e8188f3c8a5c2f73fb4e35d01032e8dc258835ec876d52c8ad8ee3d24b2fc5
+        dlOutput: ghc-9.8.4-src.tar.xz
+        dlSubdir: ghc-9.8.4
+        dlUri: https://downloads.haskell.org/~ghc/9.8.4/ghc-9.8.4-src.tar.xz
+      viTags:
+      - Latest
```

Comment:
It's not the latest

Reply:
Sigh, yes, I will drop this from the generation script.